contracts/linear/src/validator_pool.rs

```diff
@@ -499,6 +498,7 @@ impl LiquidStakingContract {
         // 2. has base stake amount set to 0
         // 3. not in pending release
         // 4. has not unstaked balance (because this part is from user's unstake request)
+        // 5. not in draining process
```

Comment:
I think we can either remove this comment, or still keep the check (though it's logically redundant, I think it still makes sense).